packages/common/src/HttpClient.ts

```diff
@@ -44,4 +46,11 @@ export class HttpClient {
     this.logger.info(`auditTransaction response: ${JSON.stringify(auditResponse)}`)
     return auditResponse
   }
+
+  async getNetworkConfiguration (): Promise<ConfigResponse> {
+    const gsnUrl = `https://raw.githubusercontent.com/opengsn/gsn-networks/${gsnRuntimeVersion}/client-config.json`
```

Comment:
Also, very very very strongly opposed to a hard-coded URL in code. If something must be configurable in this world, that's a URL.